1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh

```diff
@@ -213,6 +213,9 @@ pgbouncer_initialize() {
         if ! is_empty_value "$PGBOUNCER_AUTH_USER"; then
             database_value+=" auth_user=$PGBOUNCER_AUTH_USER"
         fi
+        if ! is_empty_value "$PGBOUNCER_SET_CONNECT_QUERY"; then
```

Comment:
Using `PGBOUNCER_SET_CONNECT_QUERY` as the name for the env var gives the idea of a boolean, just like `PGBOUNCER_SET_DATABASE_USER` (which allowed values are `yes`/`no`).

I rather use `PGBOUNCER_CONNECT_QUERY`:
```suggestion
        if ! is_empty_value "$PGBOUNCER_CONNECT_QUERY"; then
```